org.eclipse.xtext.tests/src/org/eclipse/xtext/formatting2/regionaccess/internal/RegionAccessCommentsTest.xtend

```diff
@@ -0,0 +1,261 @@
+/*******************************************************************************
+ * Copyright (c) 2017 TypeFox GmbH (http://www.typefox.io) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.formatting2.regionaccess.internal
+
+import com.google.inject.Inject
+import org.eclipse.xtext.formatting2.regionaccess.internal.regionaccesstestlanguage.RegionaccesstestlanguagePackage
+import org.eclipse.xtext.formatting2.regionaccess.internal.tests.RegionAccessTestLanguageInjectorProvider
+import org.eclipse.xtext.testing.InjectWith
+import org.eclipse.xtext.testing.XtextRunner
+import org.junit.Test
+import org.junit.runner.RunWith
+
+/**
+ * @author Moritz Eysholdt - Initial contribution and API
+ */
+@InjectWith(RegionAccessTestLanguageInjectorProvider)
+@RunWith(XtextRunner)
+class RegionAccessCommentsTest {
```

Comment:
That was the fix each time, I have stumbled upon such a Windows line ending issue. And I have to create a utility class for each project. I know it is kind of off-topic, but wouldn't it make sense to introduce such a helper in Xtext?